assets/app/scripts/controllers/pod.js

```diff
@@ -97,12 +97,17 @@ angular.module('openshiftConsole')
           logs: [],
           logsLoading: true,
           canShowDownload: false,
-          canInitAgain: false,
-          options: {
-            container: $scope.pod.spec.containers[0].name
-          }
+          canInitAgain: false
         });
 
+        if (!$scope.options || !$scope.options.container) {
+          angular.extend($scope, {
+            options: {
+              container: $scope.pod.spec.containers[0].name
+            }
+          });
+        }
```

Comment:
Is it possible to initialize this once when we call DataService.get() for the pod?